.golangci.yaml

```diff
@@ -138,7 +138,7 @@ linters:
     - errname # checks that sentinel errors are prefixed with the Err and error types are suffixed with the Error
     - errorlint # finds code that will cause problems with the error wrapping scheme introduced in Go 1.13
     - exhaustive # checks exhaustiveness of enum switch statements
-    - exportloopref # checks for pointers to enclosing loop variables
```

Comment:
We still target 1.21, so this should still be enabled

Reply:
Ci is `1.22` so the lint setup fails.  `copyloopvar` is commented out because it requires `1.22`.  
Two options:
1. uncomment `copyloopvar`
2. move CI to 1.21